tools/src/main/python/dlpx/virtualization/_internal/validation_schemas/plugin_importer.yaml

```diff
@@ -140,15 +140,15 @@ EXPECTED_DIRECT_ARGS_BY_OP:
       - repository
 
 EXPECTED_UPGRADE_ARGS:
-  repository_id_to_impl:
```

Comment:
Could you also add couple of Lua migrations to test/python/dlpx/virtualization/_internal/fake_plugin/direct so that we have coverage there as well?

Reply:
I added them for the basic errors that we had tested for platform migrations. I wasn't able to do the tests for what happens if the migration_type was either not the expected enum or if they passed in a none existing enum.

Will file a bug for that.

Reply:
https://github.com/delphix/virtualization-sdk/issues/118